packages/api/src/brevo-contact/dto/subscribe-response.enum.ts

```diff
@@ -4,6 +4,7 @@ export enum SubscribeResponse {
     SUCCESSFUL = "SUCCESSFUL",
     ERROR_UNKNOWN = "ERROR_UNKNOWN",
     ERROR_CONTAINED_IN_ECG_RTR_LIST = "ERROR_CONTAINED_IN_ECG_RTR_LIST",
+    ERROR_MAXIMAL_NUMBER_OF_CONTACTS_REACHED = "ERROR_MAXIMAL_NUMBER_OF_CONTACTS_REACHED",
```

Comment:
I would suggest renaming to `ERROR_MAXIMAL_NUMBER_OF_TEST_CONTACTS_REACHED`
since contacts are not limited, only test contacts

Reply:
8dc01a15f07bc9ff971035140c2ce4a6cae17962